easybuild/easyblocks/g/gromacs.py

```diff
@@ -468,18 +468,29 @@ def sanity_check_step(self):
             libnames.extend([libname + mpisuff for libname in libnames])
 
         suff = ''
-        # add the _d suffix to the suffix, in case of the double precission
-        if re.search('DGMX_DOUBLE=(ON|YES|TRUE|Y|[1-9])', self.cfg['configopts'], re.I):
-            suff = '_d'
 
-        libs = ['lib%s%s.%s' % (libname, suff, self.libext) for libname in libnames]
+        # make sure that configopts is a list:
+        configopts = self.cfg['configopts']
+        if type(configopts) is str:
```

Comment:
Please use `isinstance` here. 
Also, I would use a more explicit name for the list of `configopts` values:

```python
configopts_list = self.cfg['configopts']
if isinstance(configopts_list, basestring):
    configopts_list = [configopts_list]